docs/api-reference/geo-layers/mvt-layer.md

```diff
@@ -117,6 +117,11 @@ On top of the [default options](/docs/api-reference/core/layer.md#loadoptions),
 
 - [MVTLoader](https://loaders.gl/modules/mvt/docs/api-reference/mvt-loader)
 
+##### `binary` (Boolean, optional)
+
+* Default: true
+
+Reads data in binary format to improve performance. More information in the [performance optimization guide](/docs/developer-guide/performance.md#use-binary-data).
```

Comment:
The linked documentation doesn't contain anything specific to the MVTLayer. I think the important thing to document here is whether it affects any user-facing APIs (e.g. will GeoJSON feature still be available via callbacks?)

Reply:
If I just set `loadOptions: {gis: { format: 'binary' }}` will the layer auto-detect that the input is binary and do the right thing? Or I also have to set `binary` flag here?

Reply:
The GeoJSON layer auto-detect the input no flag required there.

Agree with @Pessimistress. I'll start adding the limitations in the next commits. A current know limitation is that binary mode is only supported if renderSubLayers is a GeoJSONLayer (the default).